### ISSUE TITLE
Adding "bootstrap" fails to find files.

### ISSUE THREAD

**Issue by Vaccano** [aurelia/cli#219]
1. Created a new project using `au new` named `cli-test`
2. Used options: 2 (typescript), 1 (create), 1 (get dependencies)
3. Changed Directory to the new `cli-test` folder.
4. Ran `npm install bootstrap --save`
     It gave the following error:  
`UNMET PEER DEPENDENCY Typescript@1.9.0-dev.20160627-1.0`  
     This seemed odd to me as I assumed that the `au new` would have gotten the correct typescript stuff...  (Package .json seems to want typescript 1.9 or greater but only in dev dependencies.)
5. Opened Aurelia.json and added `"bootstrap",` to bundles->vender-bundle->dependencies.
6. Ran `au run`.  

Result: I got the following errors:

```
Tracing aurelia-dependency-injection...
Tracing bootstrap...
{ uid: 9,
  name: 'writeBundles',
  branch: false,
  error:
   { [Error: ENOENT: no such file or directory, open 'C:\src\spikes\js\transition.js']
     errno: -4058,
     code: 'ENOENT',
     syscall: 'open',
     path: 'C:\\src\\spikes\\js\\transition.js',
     moduleTree: [ 'bootstrap' ],
     fileName: 'C:/src/spikes/AureliaCli2/cli-test/node_modules/bootstrap/dist/js/npm.js' },
  duration: [ 0, 175460446 ],
  time: 1467758106903 }
{ uid: 1,
  name: '<series>',
  branch: true,
  error:
   { [Error: ENOENT: no such file or directory, open 'C:\src\spikes\js\transition.js']
     errno: -4058,
     code: 'ENOENT',
     syscall: 'open',
     path: 'C:\\src\\spikes\\js\\transition.js',
     moduleTree: [ 'bootstrap' ],
     fileName: 'C:/src/spikes/AureliaCli2/cli-test/node_modules/bootstrap/dist/js/npm.js' },
  duration: [ 1, 908511472 ],
  time: 1467758106917 }
{ uid: 0,
  name: '<series>',
  branch: true,
  error:
   { [Error: ENOENT: no such file or directory, open 'C:\src\spikes\js\transition.js']
     errno: -4058,
     code: 'ENOENT',
     syscall: 'open',
     path: 'C:\\src\\spikes\\js\\transition.js',
     moduleTree: [ 'bootstrap' ],
     fileName: 'C:/src/spikes/AureliaCli2/cli-test/node_modules/bootstrap/dist/js/npm.js' },
  duration: [ 1, 927638957 ],
  time: 1467758106935 }
{ [Error: ENOENT: no such file or directory, open 'C:\src\spikes\js\transition.js']
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\src\\spikes\\js\\transition.js',
  moduleTree: [ 'bootstrap' ],
  fileName: 'C:/src/spikes/AureliaCli2/cli-test/node_modules/bootstrap/dist/js/npm.js' }
```

These steps seem to be what is asked for in the "[Adding Client Libraries to Your Project](https://github.com/aurelia/cli#adding-client-libraries-to-your-project)" section of the readme.md.

**Comment by doktordirk** [comment]
+1

**Comment by MAAARKIN** [comment]
+1

**Comment by TylerJPresley** [comment]
```
"bundles": [
      {
        "name": "app.js",
        "source": [
          "[**/*.js]",
          "**/*.html"
        ],
        "dependencies": [
          "jquery",
          "toastr",
          "moment",
          "nprogress",
          "clone",
          "numeral",
          "store",
          "bootstrap-datepicker",
          {
            "name": "chart.js",
            "path": "../node_modules/chart.js/dist/",
            "main": "Chart.bundle"
          },
          {
            "name": "bootstrap",
            "path": "../node_modules/bootstrap/dist/js",
            "main": "bootstrap"
          }
        ]
      },
```

**Comment by MAAARKIN** [comment]
@TylerJPresley but i want to put css files, this way add css too?

**Comment by Vheissu** [comment]
A cool tip for anyone who is using Sass in their CLI apps. If you install `bootstrap-sass` you can import the Sass stylesheet from the `node_modules/bootstrap-sass` plugin and it'll bundle in the Bootstrap CSS for you without needing to use the bundle stuff.

**Comment by doktordirk** [comment]
@MAAARKIN 
meanwhile add
```html
    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
```
to the head of the index.html

@EisenbergEffect 
it is actually currently possible to sortof import the bootstrap.css. only the "!' isn't respected and a ".js" extension always added. so, the fix might be small!?

**Comment by EisenbergEffect** [comment]
This is now possible with CLI 0.16.0.

I'm writing an update now for the readme that explains configuring 3rd party libraries.

If you can't wait another hour or two, I've updated the `app-contacts` sample to be based on the CLI. It uses jQuery, Bootstrap and NProgress.

**Comment by beeterry** [comment]
I'm still having issues with this using AU version 0.17.0. I have a fully operable project that I created using the skeleton and VSCode that I'm trying to port over to ASP.NET Core and VS2015. I created the project via `au new --here` and I'm trying to bring it all together 1 page at a time. However I'm struggling to get the bootstrap css to work. Anytime I try to import the css from html, I get a 404 and another error that reads "Bootstrap's javascript requires jquery." I've created a detailed thread on StackOverflow. If someone could help me out, I'd appreciate it.

http://stackoverflow.com/questions/38852369/how-to-import-packages-within-my-aurelia-application
